marshmallow/fields.py

```diff
@@ -1106,13 +1106,18 @@ class Url(ValidatedField, String):
     """
     default_error_messages = {'invalid': 'Not a valid URL.'}
 
-    def __init__(self, relative=False, **kwargs):
+    def __init__(self, relative=False, schemes=None, **kwargs):
         String.__init__(self, **kwargs)
+
+        if schemes is None:
```

Comment:
These two lines should be unnecessary; `validate.URL` accepts `schemes=None`, in which case the `default_schemes` are used.